mapper/v04/alerts.go

```diff
@@ -0,0 +1,95 @@
+package v04
```

Comment:
Might be useful to add a package comment to describe the package rather than doing that on the private `alert` type later.